### ISSUE TITLE
Should the DB synonym use MonadUnliftIO?

### ISSUE THREAD

**Issue by MaxGabriel** [yesodweb/yesod-scaffold#191]
This would allow using MonadUnliftIO functions like `catch` inside `DB` code, which seems reasonable.

The definition is currently:

```
-- | A convenient synonym for database access functions.
type DB a = forall (m :: * -> *).
    (MonadIO m) => ReaderT SqlBackend m a
```

**Comment by snoyberg** [comment]
Sounds reasonable to me. Want to make the change?

**Comment by MaxGabriel** [comment]
Sure

**Comment by snoyberg** [comment]
Fixed by #192.
